entities/params/requestbody.go

```diff
@@ -0,0 +1,8 @@
+package params
+
+import "github.com/go-playground/validator/v10"
+
+type RequestBody interface {
+	trimSpaces()
+	Validate(v validator.Validate) (string, bool)
```

Comment:
The RequestBody does not need a Validate method, we could have a generic function instead, that takes a request body and performs Validate on it with the validator.